### PR TITLE
network/netdev: do not try to update several parameters if the interface already exists

### PR DESCRIPTION
**Pull request by yuwata**
split-out of #34909.

### CODE REVIEW

File: src/network/netdev/vxlan.c [hunk 1]
```diff
@@ -99,6 +93,34 @@ static int netdev_vxlan_fill_message_create(NetDev *netdev, Link *link, sd_netli
...
+        if (v->df != _NETDEV_VXLAN_DF_INVALID) {
+                r = sd_netlink_message_append_u8(m, IFLA_VXLAN_DF, v->df);
+                if (r < 0)
+                        return r;
+        }
+
+        if (netdev->ifindex > 0)
+                return 0;
+
+        /* The properties below cannot be updated, and the kernel refuse the whole request if one of the
```
Comment (poettering) [inline src/network/netdev/vxlan.c]: refuse → refuses

File: src/network/netdev/macsec.c [hunk 2]
```diff
@@ -515,6 +509,17 @@ static int netdev_macsec_fill_message_create(NetDev *netdev, Link *link, sd_netl
         if (r < 0)
                 return r;
 
+        /* The properties below cannot be updated, and the kernel refuse the whole request if one of the
```
Comment (poettering) [inline src/network/netdev/macsec.c]: ditto

**Comment by poettering** [comment]
no test case?

**Comment by yuwata** [comment]
> no test case?

Waiting in #34909. I do not want to add it in this PR.

**Pull request merged**